### PR TITLE
2196242 bosh documentation

### PR DESCRIPTION
**Pull request by gogolok**

### CODE REVIEW

File: docs-content/usecase.html.md.erb [hunk 1]
```diff
@@ -17,8 +17,54 @@ etc.
...
 the corresponding a9s service tile will talk to the <b>a9s BOSH for PCF</b> tile and create a BOSH deployment in order to provision virtual machines and to install the required software packages.
 
-## Why we decided not to use the PCF Ops Manager Director for this purpose
+<img src="./images/a9s-bosh-for-pcf-architecture.png" alt="a9s BOSH for PCF Architecture" />
 
-We want to provide the possiblity to choose between the <b>a9s BOSH for PCF</b> and the <b>PCF Op Manager Director</b> in the future. For proof of concepts this would be a useful option, because the <b>a9s BOSH for PCF</b> tile does not need to be installed.
+## Advantages of Using a9s BOSH for PCF
 
-Due to the missing experience with using one BOSH director only for the regular PCF tile deployments as well as for the service instance deployments, we focus our support on using the a9s BOSH for PCF tile for now.
+In general, it is expected to have only a few tiles in your <b>PCF Ops Manager Director</b>,
```
Comment (lucaspinto) [inline docs-content/usecase.html.md.erb]: Not a huge fan of the "this is expected" formulation. What do you mean here? Are you describing a typical use case?

File: docs-content/usecase.html.md.erb [hunk 2]
```diff
@@ -17,8 +17,54 @@ etc.
...
+this could lead to a series of problems:
+
+* __Bottleneck:__ the <b>PCF Ops Manager Director</b> might take too long to
+execute a task for the Data Service because it is handling tasks from thousands
+of service instances.
+
+* __Upgrade:__ if something goes wrong when upgrading the <b>PCF Ops Manager Director</b>
+the service instances could be affected.
+
+* __Configuration:__ the <b>Ops Manager</b> might not provide a configuration
```
Comment (lucaspinto) [inline docs-content/usecase.html.md.erb]: The way to paragraph is written is quite confusing. It feels that this needs to be cut in at least 2 sentences. It also has grammar mistakes :)

File: docs-content/usecase.html.md.erb [hunk 3]
```diff
@@ -17,8 +17,54 @@ etc.
...
+option necessary and to change it manually is not a safe option, since to apply
+changes using the dashboard overrides the current configuration.
+
+* __Confusing Environment:__ the instances created by the service shares the same space
+as the tiles, however the <b>Ops Manager</b> does not reflect this properly.
+
+* Etc.
```
Comment (lucaspinto) [inline docs-content/usecase.html.md.erb]: Say that the list is not exhaustive if needed, but avoid "Etc." in bullet points.

File: docs-content/usecase.html.md.erb [hunk 4]
```diff
@@ -17,8 +17,54 @@ etc.
...
+handle specially the service instances. This way applying changes to <b>PCF Ops
+Manager Director</b> can be done without the possibility to lose data.
+
+* __Source of Truth:__ the service instances are always in <b>a9s BOSH for PCF</b>
+and the services are always in <b>PCF Ops Manager Director</b>, there is no hidden
+or mixed releases, deployments, instances etc.
+
+* __IaaS Portability:__ a complete different IaaS to provide service instances.
+
+* Etc.
```
Comment (lucaspinto) [inline docs-content/usecase.html.md.erb]: It is possible to have "Etc." ? :)

**Review by lucaspinto — CHANGES_REQUESTED** [review]
Let's talk about this together and see how we can rewrite some of those.

**Comment by phartz** [comment]
close pull against our master branch
I updated the repo with the upstream and will merge into a new feature, which we can use to create our PR against the forked repo from pivotal

**Pull request closed without merging**